samcli/lib/utils/hash.py

```diff
@@ -3,9 +3,21 @@
 """
 import hashlib
 import os
+import sys
 from typing import Any, List, Optional, cast
 
 BLOCK_SIZE = 4096
+# earliest python version to support usedforsecurity option for hashlib.md5 is 3.9
+# https://docs.python.org/3/library/hashlib.html#hash-algorithms
+_MAJOR_PYTHON_VERSION = 3
+_MINOR_PYTHON_VERSION = 9
```

Comment:
question - Should we bump the python version we used to building installers?

Reply:
Most likely. In the ideal world, we are always on the latest stable version. It will cause less headaches for us in the long term for various reason. @mndeveci and I chatted about this, we likely need a task in our backlog.